lib/LedgerSMB/Database/Loadorder.pm

```diff
@@ -69,7 +69,7 @@ sub scripts {
     my @scripts =
         map { $self->_process_script($_)}
         grep { $_ =~ /\S/ }
-        map { my $string = $_; $string =~ s/#.*$//; $string }
+        map { my $string = $_; $string =~ s/\s*#.*$//; $string }
```

Comment:
Is this solving a problem related to this PR? it allows comments to be prepended with spaces, so it changes the semantics of the LOADORDER file. Is that required for this pr?

Reply:
Ok. But what makes it required to make this change?

Reply:
Some errors are generated because trailing spaces aren't currently stripped and lines do not match.